packages/react-components/react-color-picker-preview/library/docs/Spec.md

```diff
@@ -132,51 +130,18 @@ For custom sizes users might want to customize it via CSS.
 
 ### ColorPicker
 
-| Property   | Values              | Default   | Purpose                                |
-| ---------- | ------------------- | --------- | -------------------------------------- |
-| color      | `HSV`               |           | Sets color value in HSV format         |
-| onChange   | `function`          | undefined | Callback called when color is selected |
-| shape      | `square`, `rounded` | `rounded` | Sets shape                             |
-| step       | `number`            | 1         | Step for arrow navigation              |
-| customStep | `number`            | 3         | Fast navigation                        |
```

Comment:
Those are supported by default. And `customStep` idea was for faster navigation but because we use native inputs it's supported from the box.